backend/test/game/test_turn.py

```diff
@@ -0,0 +1,54 @@
+import collections
+import pytest
+
+import backend.src.game.turn as turn
+from backend.src.game.dice.dice import DieValue
+
+
+def test_sub_three_victory_point_dice():
```

Comment:
Sorry to be anal, but it's my job yay.
Please make sure test names describe system behavior as well as system input.
For example:
test_less_than_three_victory_point_dice_no_victory_points_awarded